components/NewsletterHome.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useRuntimeConfig } from '#app'
+import axios from 'axios'
```

Comment:
It seems like overkill to include all of axios just for the newsletter, I think we cant probably hand the form submission with a regular fetch.